R/plots.R

```diff
@@ -83,6 +88,41 @@ plotStudy <- function(study, modelID, featureID, plotID, testID = NULL, librarie
         sprintf("Received %d testID(s)", nTests)
       )
     }
+    # multiModel is checked as a multiTest as it requires at least 2 testIDs, eg.:
+    # (1) 1 testID per model and > 1 model
+    # (2) > 1 testID and 1 model
+    if (plotType[ind] == "multiModel") {
+      if (nTests < 2) {
+        stop(
+          "Plot type \"multiModel\" requires at least 2 testIDs\n",
+          sprintf("Received the following testID(s) : %s ", testID)
+        )
+      }
+      if ((!is.null(names(testID)) & any(is.na(names(testID)))) | is.null(names(testID))) {
+        stop(
+          "Plot type \"multiModel\" requires a vector for testID named after related modelID"
+        )
+      }
+      if (nModels > 1) {
+        mapping <- getMapping(study = study, libraries = libraries)
+        if (is.list(mapping) & length(mapping) == 0) {
+          stop(
+            "Plot type \"multiModel\" requires mapping object if > 1 modelID is used\n",
```

Comment:
Could you please add a unit test for this error? You can create a new study object that doesn't have `mapping`, e.g.

```R
testStudyObjNoMapping <- testStudyObj
testStudyObjNoMapping[["mapping"]] <- list()
```